src/main/java/hu/dungeonhunter/characters/Champion.java

```diff
@@ -0,0 +1,35 @@
+package hu.dungeonhunter.characters;
+import hu.dungeonhunter.tools.Dice;
+
+public class Champion {
+    static final int START_HP = 20;
+    private int championHP;
+    private final int championMaxDamage;
+    private int championNumOfDices;
+
+    public Champion() {
+        this.championHP = START_HP;
+        this.championMaxDamage = 6;
+        this.championNumOfDices = 1;
```

Comment:
To print champion data, you can do this here:
```
System.out.println("Your champion have " + champion.getChampionHP() + " HP and can do " +
                champion.getNumOfDices() + "-" + champion.getNumOfDices() * champion.getChampionMaxDamage() +
                " damage.");
```

So this will be printend automatically when you say new Champion(). Whatever is inside the class initializer method will be executed when the class is instantiated.

Reply:
I did, but dont working, all way give back 0 value.

after run, print this:
"Your champion have 0 HP and can do 0-0 damage."

Reply:
Yes, my bad. Getters won't work here yet, because we're not finsihed with the initialization.
Try this:
`        System.out.println("Your champion have " + championHP + " HP and can do " +
            championNumOfDices + "-" + championNumOfDices * championMaxDamage +
            " damage.");`